addons/binding/org.openhab.binding.wso2iots/.classpath

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<classpath>
+	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.8"/>
+	<classpathentry kind="con" path="org.eclipse.pde.core.requiredPlugins"/>
+	<classpathentry kind="src" path="src/main/java"/>
+	<classpathentry kind="lib" path="/home/rameshakarunasena/Downloads/gson-2.6.2.jar"/>
```

Comment:
This seems local configuration?

Reply:
Also, GSON is provided in the targetplatform.